### ISSUE TITLE
ORT (Docker built) Analyzer exits without error after printing environment variables

### ISSUE THREAD

**Issue by daniel-eder** [oss-review-toolkit/ort#3035]
# Description
When running the analyzer with a custom docker image built from commit https://github.com/oss-review-toolkit/ort/commit/9759e9056a07f9f3ad9401cd745e29edaf920ab8 it runs and correctly prints out the environment variables:
```
More environment variables:
ANDROID_HOME = /opt/android-sdk
GOPATH = /go
```
But then exits without any further output or error.

# Details 
- Target repository: https://github.com/elastic/kibana/ (tested with https://github.com/elastic/kibana/commit/19bf9ed072ca15157e11671745e3783efe27d925 and later)
- This reproduces on Ubuntu 16.04 and 20.04 (no other versions tested), but does not reproduce on Windows 10.
- The commands used were `sudo docker run danieleder/ort:9759e90 -v $(pwd):/project --info analyze -i /project -o /project/ort/analyzer` and `sudo docker run danieleder/ort:9759e90 -v $(pwd):/project -v $(pwd)/../ort-results:/ort --info analyze -i /project -o /ort/analyzer` (output directory separate from scanned project), both yield the same result.  
- Hardware should be sufficient, tested with 8-16 GB RAM and 2-12 vCPUs.
- Tested on azure VMs and local VirtualBox VMs

EDIT: This also reproduces on Fedora 32 in a VirtualBox VM

**Comment by sschuberth** [comment]
It tool even myself a while to find out, but you've simply swapped the `-v` option for `docker run` (for *volume*) with the `-v` option for ORT (for *version*), which is why ORT just exits after printing its version when it sees `-v`. The correct option order would be

    sudo docker run -v $(pwd):/project danieleder/ort:9759e90 --info analyze -i /project -o /project/ort/analyzer

**Comment by daniel-eder** [comment]
Oh gosh - mea culpa. Worst case of being blind to the obvious, truly sorry for the mistaken report. Teaches me to properly verify my test data! 

That elegantly explains why it worked on windows (I used my own existing command, with the poweshell variable syntax for the current directory, and incidentally also the correct order) but not on Linux (I used our developer's command with the Linux variable syntax for the current directory and switched order), because a direct copy paste would not work due to the current directory variable.
